fix for git issue 4705

davidmu1 opened the pull request.

opbld33 commented:
Docs Build status updates of commit _[c288189](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4774/commits/c28818987459d7b00712cf68576cc4b0188da07f)_: 

:clock10: **Pending**: waiting for processors ([7](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 7 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

opbld33 commented:
Docs Build status updates of commit _[c288189](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4774/commits/c28818987459d7b00712cf68576cc4b0188da07f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/user-post-users.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/gitissue4705-2/api-reference/beta/api/user-post-users.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/user-post-users?view=graph-rest-beta&branch=pr-en-us-4774) (graph-rest-beta) |
[api-reference/beta/resources/invitation.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/gitissue4705-2/api-reference/beta/resources/invitation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/invitation?view=graph-rest-beta&branch=pr-en-us-4774) (graph-rest-beta) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C7%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201906071637276163-4774%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=5qo65L8XrAPswl81tERYTZeKWjkcXTm5F7%2BIB2elQpo%3D&st=2019-06-07T16%3A35%3A34Z&se=2019-07-08T16%3A40%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

davidmu1 commented:
The GitHub issue asked to add a link to the invitation article to make developers aware of the way to create an external user account. Changed the title of the invitation article. Added a sentence saying that to create an external user, an invitation must be used.

#sign-off

Lauragra commented on api-reference/beta/resources/invitation.md:
```diff
@@ -1,16 +1,16 @@
 ---
-title: "invitation manager"
-description: "Use the invitation manager to create an invite, in order to add an external user to the organization. "
+title: "invitation resource type"
```
@davidmu1 , I just noticed that this file is also in v1.0. Can you please make the same update to the v1.0 version of the file?

Lauragra commented on api-reference/beta/resources/invitation.md:
```diff
@@ -1,16 +1,16 @@
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Use the invitation manager to create an invite, in order to add an external user to the organization. 
+The invitation is used to create an invite, in order to add an external user to the organization. 
```
We're using a definite article here which might be unclear. Resource topics often begin with a verb like "Represents". Suggested revision: "Represents an invitation that is used to add external users to an organization."

opbld33 commented:
Docs Build status updates of commit _[e1fc3fb](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4774/commits/e1fc3fb69a04a379a00cff8f9f42a5b423c59c23)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/user-post-users.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/gitissue4705-2/api-reference/beta/api/user-post-users.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/user-post-users?view=graph-rest-beta&branch=pr-en-us-4774) (graph-rest-beta) |
[api-reference/beta/resources/invitation.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/gitissue4705-2/api-reference/beta/resources/invitation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/invitation?view=graph-rest-beta&branch=pr-en-us-4774) (graph-rest-beta) |
[api-reference/v1.0/resources/invitation.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/gitissue4705-2/api-reference/v1.0/resources/invitation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/invitation?view=graph-rest-1.0&branch=pr-en-us-4774) (>= graph-rest-1.0) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C11%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201906111553448985-4774%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=4o%2BV%2BWFFASa0A9HBEGlMLkwpvJXNfDowj6%2Br1rzJ53E%3D&st=2019-06-11T15%3A51%3A45Z&se=2019-07-12T15%3A56%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

opbld32 commented:
Docs Build status updates of commit _[72d6072](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4774/commits/72d6072e55d50849038d578e8bc34f17ee3b395a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/user-post-users.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/gitissue4705-2/api-reference/beta/api/user-post-users.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/user-post-users?view=graph-rest-beta&branch=pr-en-us-4774) (graph-rest-beta) |
[api-reference/beta/resources/invitation.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/gitissue4705-2/api-reference/beta/resources/invitation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/invitation?view=graph-rest-beta&branch=pr-en-us-4774) (graph-rest-beta) |
[api-reference/v1.0/api/user-post-users.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/gitissue4705-2/api-reference/v1.0/api/user-post-users.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/user-post-users?view=graph-rest-1.0&branch=pr-en-us-4774) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/invitation.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/gitissue4705-2/api-reference/v1.0/resources/invitation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/invitation?view=graph-rest-1.0&branch=pr-en-us-4774) (>= graph-rest-1.0) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C11%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201906111600102461-4774%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=xzzo9VkkQ%2BpEIh6CazklxuatMtbcQIl2Pm%2BOAgRGrmw%3D&st=2019-06-11T15%3A58%3A17Z&se=2019-07-12T16%3A03%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

opbld32 commented:
Docs Build status updates of commit _[ecbbcbc](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4774/commits/ecbbcbcf6379cfba004b057fd89e1867cba49c8a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/user-post-users.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/gitissue4705-2/api-reference/beta/api/user-post-users.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/user-post-users?view=graph-rest-beta&branch=pr-en-us-4774) (graph-rest-beta) |
[api-reference/beta/resources/invitation.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/gitissue4705-2/api-reference/beta/resources/invitation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/invitation?view=graph-rest-beta&branch=pr-en-us-4774) (graph-rest-beta) |
[api-reference/v1.0/api/user-post-users.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/gitissue4705-2/api-reference/v1.0/api/user-post-users.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/user-post-users?view=graph-rest-1.0&branch=pr-en-us-4774) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/invitation.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/gitissue4705-2/api-reference/v1.0/resources/invitation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/invitation?view=graph-rest-1.0&branch=pr-en-us-4774) (>= graph-rest-1.0) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C11%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201906111622490606-4774%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=csubeOmXg5PVuAcJ1ln0W%2B%2Fsfk4eh9mlr5vbeo0iHS8%3D&st=2019-06-11T16%3A20%3A42Z&se=2019-07-12T16%3A25%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.
